ansible/roles/config-idm-server/tasks/configure_idm.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Setup hostname entry
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ private_ip }} {{ idm_dns_name }}"
+    state: present
+     
+- name: Configure initial IdM setup
+  command: >
+    ipa-server-install -U
+      --hostname="{{ idm_dns_name | default(ansible_fqdn) }}"
+      --domain="{{ idm_domain }}"
+      --realm="{{ idm_realm | upper }}"
+      --ds-password="{{ idm_dm_password }}"
+      --admin-password="{{ idm_admin_password }}"
+      --no-host-dns
+  ignore_errors: true
+  when:
+  - idm_src is not defined
+
+- name: Add IdM client for replica
+  command: >
+    ipa-client-install -U
+      --domain="{{ idm_domain }}"
+      --server="{{ idm_src }}"
+      --principal="{{ idm_principal }}"
+      --password="{{ idm_admin_password }}"
+      --force-join
+  when:
+  - idm_src is defined
+
+- name: Install and Configure IdM replica
+  command: >
+    ipa-replica-install -U
+      --principal="{{ idm_principal }}"
+      --admin-password="{{ idm_admin_password }}"
+  ignore_errors: true
```

Comment:
why `ignore_errors` ?

Reply:
I am re-using code from Redhat-cop/infra-ansible here.  I imagine they had the same problems with isa-replica-install due to lack of idempotency and it possibly falsely reporting that it failed.  I can remove the capability to set up replication if we so desire.

Reply:
Remove the replica code as it is unused.